sim/firesim-lib/src/main/cc/bridges/uart.h

```diff
@@ -15,28 +17,68 @@
 // all-caps, suffixed with "_struct" and "_struct_guard" respectively.
 
 #ifdef UARTBRIDGEMODULE_struct_guard
-class uart_t : public bridge_driver_t {
+
+/**
+ * Base class for callbacks handling data coming in our out a UART stream.
```

Comment:
```suggestion
 * Base class for callbacks handling data coming in and out of our UART stream.
```
This is what you meant right?